app/controllers/admin_scope/reports_controller.rb

```diff
@@ -1,8 +1,8 @@
 module AdminScope
   class ReportsController < BaseController
     expose :company, -> { current_user.own_company }
-    expose :authors, -> { set_authors }
-    expose :filtered_authors, -> { filter_authors }
+    expose :authors, -> { raw_authors.sorted_by_comments_count }
+    expose :filtered_authors, -> { filtered_authors.sorted_by_comments_count }
```

Comment:
not good using the same names
```suggestion
    expose :authors, -> { filtered_authors.sorted_by_comments_count }
```

Reply:
also why we need `expose :authors` and `expose :filtered_authors`?